suricata.yaml.in

```diff
@@ -112,6 +112,12 @@ outputs:
 
       # Include top level metadata. Default yes.
       #metadata: no
+      # Include stream data in application layer events
+      #stream-data: yes
+      #stream-data-printable: yes
+      # Full log for alerted flow. Maximize verbosity of event when the flow
+      # has alerts. Disabled by default. Possible value no, printable, base64, both.
+      #full-log-alerted: printable
```

Comment:
I don't understand the difference between this and `stream-data` and `stream-data-printable`. Both seem to set up the same `include_streamdata` field.

Reply:
Yes, one is triggering printable format (`stream-data-printable`) and the other one base64 encoding (`stream-data`).